ChatSX/screens/HomeFeed/index.tsx

```diff
@@ -154,6 +158,31 @@ const HomeFeed = ({navigation}: Props) => {
     fetchData();
   }, [navigation, myClient]);
 
+  useEffect(() => {
+    const subscription = Linking.addEventListener('url', ({url}) => {
```

Comment:
why `subscription`, can we name the variable as `listener` as it is `Linking.addEventListener` is returning a `listener`